Source/Core/DolphinWX/MemcardManager.cpp

```diff
@@ -124,7 +124,7 @@ CMemcardManager::CMemcardManager(wxWindow* parent, wxWindowID id, const wxString
 		mcmSettings.usePages = true;
 		for (int i = COLUMN_BANNER; i < NUMBER_OF_COLUMN; i++)
 		{
-			mcmSettings.column[i] = (i <= COLUMN_FIRSTBLOCK)? true:false;
+			mcmSettings.column[i] = (i <= COLUMN_FIRSTBLOCK) ? true : false;
```

Comment:
You could change this to `!!(i <= COLUMN_FIRSTBLOCK)` but I understand if you don't want to do it in this PR...